state-chain/primitives/src/lib.rs

```diff
@@ -151,6 +151,9 @@ pub struct TxId {
 /// The very first epoch number
 pub const GENESIS_EPOCH: u32 = 1;
 
+/// Number of blocks in the future a swap is scheduled for.
+pub const SWAP_DELAY_BLOCKS: u32 = 2;
```

Comment:
I'm not a fan of putting configuration constants in the primitives...

Reply:
We don't really have a place for sharing constants between crates/pallets. Is it worth creating a separate crate for just constants?

Reply:
Alternatively, if the issue is just about the name, maybe we could rename "primitives" to "common"?

Reply:
Let's not rename the whole module just yet 😅